terraform/README.adoc

```diff
@@ -71,7 +86,9 @@ users.users.gila = {
 === Setting up LXD on Nixos
 
 Using LXD is faster and requires fewer resources. Also, you can do many things
-with lxd that you typically can't do quickly or easily with VMs.
+with lxd that you typically can't do quickly or easily with VMs. Make sure
+that your system is using *unstable channel* for nixpkgs (at least LXD v4 is
```

Comment:
I was able to use LXD v3, on Debian 10 because that's the latest version that can be installed due to its older version of snap.

Reply:
For NixOS the requirement is LXD v4. I suppose it's not related to LXD itself, but rather to improvements done to packaging scripts in the nix. This section is about setting up LXD on NixOS, but if you agree I can add a note to the following section that is about LXD on other distros that a v3 is sufficient. If you confirm that you agree I will make the change. Thanks

Reply:
Yes, that's fine.